tests/articulation_tests.ts

```diff
@@ -68,10 +68,10 @@ function drawArticulations(options: TestOptions): void {
     f.StaveNote({ keys: ['c/4'], duration: 'q', stem_direction: 1 }),
     f.StaveNote({ keys: ['a/4'], duration: 'q', stem_direction: 1 }),
   ];
-  notesBar1[0].addModifier(0, new Articulation(sym1).setPosition(4));
-  notesBar1[1].addModifier(0, new Articulation(sym1).setPosition(4));
-  notesBar1[2].addModifier(0, new Articulation(sym1).setPosition(3));
-  notesBar1[3].addModifier(0, new Articulation(sym1).setPosition(3));
+  notesBar1[0].addModifier(new Articulation(sym1).setPosition(4), 0);
+  notesBar1[1].addModifier(new Articulation(sym1).setPosition(4), 0);
+  notesBar1[2].addModifier(new Articulation(sym1).setPosition(3), 0);
+  notesBar1[3].addModifier(new Articulation(sym1).setPosition(3), 0);
```

Comment:
0s not really required with `Articulation`. I would remove all of them in this file. Up to you.